src/freertos_drivers/ti/TivaDCC.hxx

```diff
@@ -877,15 +899,36 @@ inline void TivaDCC<HW>::interrupt_handler()
             current_bit = DCC_ONE;
         }
     }
+    if (bit_repeat_count >= 4)
+    {
+        // Forces reinstalling the timing.
+        last_bit = NUM_TIMINGS;
+    }
     if (last_bit != current_bit)
     {
         auto* timing = &timings[current_bit];
-        MAP_TimerLoadSet(HW::INTERVAL_BASE, TIMER_A, timing->interval_period);
-        MAP_TimerLoadSet(HW::CCP_BASE, TIMER_A, timing->period);
-        MAP_TimerLoadSet(HW::CCP_BASE, TIMER_B, timing->period);
-        MAP_TimerMatchSet(HW::CCP_BASE, TIMER_A, timing->transition_a);
-        MAP_TimerMatchSet(HW::CCP_BASE, TIMER_B, timing->transition_b);
+        // The delta in ticks we add to each side of the signal.
```

Comment:
Somewhere in this pull request, possibly here, can you add some commentary about the spread of timing, expected number of steps, and the expected size of the steps?

Reply:
Added documentation to the class comment.